### DISCUSSION TITLE
OOT OF ADSB with GNU 3.10 and other new dependency

### DISCUSSION THREAD

**Discussion by kaledawit** [mhostetter/gr-adsb#61] [General]
Dear Thank you for your effort to share the project.I follow your instruction to install gr-adsb and its appear in the block tree.but when I execute  the following error is happen.
 Executing: /usr/bin/python3 -u /home/kale/gr-adsb/examples/adsb_rx.py

Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
gr-osmosdr 0.2.0.0 (0.2.0) gnuradio 3.10.1.1
built-in source types: file fcd rtl rtl_tcp uhd hackrf bladerf rfspace airspy airspyhf soapy redpitaya freesrp 
Using HackRF One with firmware 2023.01.1
Traceback (most recent call last):
  File "/home/kale/gr-adsb/examples/adsb_rx.py", line 271, in <module>
    main()
  File "/home/kale/gr-adsb/examples/adsb_rx.py", line 249, in main
    tb = top_block_cls()
  File "/home/kale/gr-adsb/examples/adsb_rx.py", line 185, in __init__
    self.adsb_decoder_0 = adsb.decoder("Extended Squitter Only", "None", "Brief")
  File "/usr/local/lib/python3.10/dist-packages/gnuradio/adsb/decoder.py", line 292, in __init__
    self.screen = curses.initscr()
  File "/usr/lib/python3.10/curses/__init__.py", line 29, in initscr
    setupterm(term=_os.environ.get("TERM", "unknown"),
_curses.error: setupterm: could not find terminal
Exception ignored in atexit callback: <built-in function endwin>
_curses.error: must call initscr() first

Any suggestion????

**Comment by darrell-barabash** [comment]
Did you ever solve this @kaledawit? 

I'm having the exact same issue. I tried setting TERMINFO=/usr/lib/terminfo and TERM=xterm-256color in my .bashrc and that's not enough.

**Comment by darrell-barabash** [comment]
[Solved] Problem seems to be that the gnuradio-companion terminal window doesn't support ncurses. If you simply run the adsb_rx.py from a command line window (terminal) it works great. No errors and properly formatted output. Seems obvious now ... ;-)

**Reply by kaledawit** [reply]
Dear @darrell-barabash  Thank you for valuable information.I will trying. But using gnu radio python command in the same working environment it works.
